egs2/fleurs/asr1/local/create_dataset.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import traceback
+
+import pandas as pd
+
+try:
+    from datasets import load_dataset
```

Comment:
This is a very good way to handle the import error.
In addition, you may also add this error handling to `local/path.sh` (See https://github.com/espnet/espnet/blob/master/egs2/how2_2000h/asr1/local/path.sh)

Reply:
This is our standard way to handle the installation issue